### PR TITLE
Frontend work station

### PR DESCRIPTION
**Pull request by rkarolis**
New article for ESLint in Windows

### CODE REVIEW

**Review by no-chris — CHANGES_REQUESTED** [review]
Nice typos corrections!  👍 
Good first draft on the new page, I have a few remarks to make it more generic

File: articles/Installation and Upgrading/setup-eslint-on-windows-workstation.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Setup Javascript work station
+
+## 1. Prepare PHP Storm
+To install ESLint plugin on PHP Storm go to: File > Settings > Plugins > Browse Repositories and search for ESLint, install and relaunch PHP Storm.
+
+Install Node.js and install ESLint globally with command `npm i -g eslint`.
```
Comment (no-chris) [inline articles/Installation and Upgrading/setup-eslint-on-windows-workstation.md]: Please add the path of the eslint config file in TAO

File: articles/Installation and Upgrading/install-from-config-file.md [hunk 2]
```diff
@@ -31,9 +34,11 @@ Example:
 ```bash
 # Normal CLI setup with detailed logs
 sudo -u www-data php tao/scripts/taoSetup.php /var/www/path/to/your/configFile.json -vvv
+php tao/scripts/taoSetup.php /path/to/your/configFile.json -vvv
```
Comment (no-chris) [inline articles/Installation and Upgrading/install-from-config-file.md]: I think it's enough with the Windows difference above, repeating the command twice can be confusing

File: articles/Installation and Upgrading/install-from-config-file.md [hunk 3]
```diff
@@ -31,9 +34,11 @@ Example:
...
 
 # Fully detailed logs with no color (recommended for build processes)
 sudo -u www-data php tao/scripts/taoSetup.php /var/www/path/to/your/configFile.json -vvvv -nc
+php tao/scripts/taoSetup.php /path/to/your/configFile.json -vvvv -nc
```
Comment (no-chris) [inline articles/Installation and Upgrading/install-from-config-file.md]: same here

**Review by no-chris — CHANGES_REQUESTED** [review]

File: articles/Installation and Upgrading/setup-eslint-on-windows-workstation.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+Configuration file is located at: `package-tao\tao\views\build\.eslintrc.json`
+
```
Comment (dietertaotesting) [inline articles/Installation and Upgrading/setup-eslint-on-windows-workstation.md]: Please use the following convention for paths:
- The root directory is the one that contains all extensions
- Use forward slashes, even when your environment is Windows
The path would then be _tao/views/build/.eslintrc.json_

**Review by ghost — COMMENTED** [review]
This is actually not bad all. Some of the code has broken formatting which is a well known problem. You are _not_ expected to fix the formatting problems, unless the changes are tiny. Wait until after the workshop to learn more about formatting before you do anything

**Review by ghost — CHANGES_REQUESTED** [review]
Make sure you checkout in Unix style
https://help.github.com/articles/dealing-with-line-endings/

**Pull request merged**